Packages.props

```diff
@@ -1,29 +1,29 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<ItemGroup>
-		<PackageReference Update="apophis.Lexer" Version="[1.4.1, 2)" />
-		<PackageReference Update="Autofac" Version="[5.1.2, 6)" />
-		<PackageReference Update="Funcky" Version="[2.1.1, 3)" />
+		<PackageReference Update="apophis.Lexer" Version="[1.4.2, 2)" />
+		<PackageReference Update="Autofac" Version="[6.1.0, 7)" />
+		<PackageReference Update="DotNet.Glob" Version="[3.1.2, 4)" />
+		<PackageReference Update="Funcky" Version="[2.3.0, 3)" />
 		<PackageReference Update="JetBrains.Annotations" Version="[2019.1.3]" />
 		<PackageReference Update="LibGit2Sharp" Version="[0.26.2, 0.27)" />
+		<PackageReference Update="Messerli.FileSystem" Version="[0.1.7, 2)" />
+		<PackageReference Update="Messerli.Test.Utility" Version="[0.8.4, 0.9)" />
+		<PackageReference Update="Microsoft.Build" Version="[16.8.0, 17)" />
+		<PackageReference Update="Microsoft.Build.Locator" Version="[1.3.2, 2)" />
+		<PackageReference Update="Microsoft.NET.Test.Sdk" Version="[16.8.3, 17)" />
 		<PackageReference Update="Moq" Version="[4.13.1, 5)" />
-		<PackageReference Update="Pastel" Version="[1.3.1, 2)" />
-		<PackageReference Update="Soltys.ChangeCase" Version="[1.6, 2)" />
-		<PackageReference Update="Stubble.Core" Version="[1.7.2, 2)" />
-		<PackageReference Update="System.CommandLine" Version="[2.0.0-beta1.20104.2, 3)" />
-		<PackageReference Update="Microsoft.NET.Test.Sdk" Version="[16.5.0, 17)" />
-		<PackageReference Update="xunit" Version="[2.4.1, 3)" />
-		<PackageReference Update="xunit.runner.visualstudio" Version="[2.4.1, 3)" />
-		<PackageReference Update="System.Collections.Immutable" Version="[1.7.0, 2)" />
-		<PackageReference Update="Messerli.FileSystem" Version="[0.1.2, 0.2)" />
-		<PackageReference Update="Messerli.Test.Utility" Version="[0.8.0, 0.9)" />
 		<PackageReference Update="Newtonsoft.Json" Version="[12.0.3, 13)" />
-		<PackageReference Update="DotNet.Glob" Version="[3.0.9, 4)" />
-		<PackageReference Update="NuGet.Commands" Version="[5.5.1, 6)" />
-		<PackageReference Update="Microsoft.Build" Version="[16.5.0, 17)" />
-		<PackageReference Update="Microsoft.Build.Locator" Version="[1.2.6, 2)" />
+		<PackageReference Update="NuGet.Commands" Version="[5.8.1, 6)" />
+		<PackageReference Update="Pastel" Version="[2.1.0, 3)" />
+		<PackageReference Update="Soltys.ChangeCase" Version="[1.6.0, 2)" />
+		<PackageReference Update="Stubble.Core" Version="[1.9.3, 2)" />
+		<PackageReference Update="System.Collections.Immutable" Version="[5.0.0, 6)" />
+		<PackageReference Update="System.CommandLine" Version="[2.0.0-beta1.20574.7 , 3)" />
+		<PackageReference Update="xunit" Version="[2.4.1, 3)" />
+		<PackageReference Update="xunit.runner.visualstudio" Version="[2.4.3, 3)" />
 	</ItemGroup>
 	<ItemGroup>
-		<PackageReference Update="Messerli.CodeStyle" Version="[1.2.0, 2)" />
+		<PackageReference Update="Messerli.CodeStyle" Version="[1.2.1, 2)" />
```

Comment:
The [2.0.0 rc](https://www.nuget.org/packages/Messerli.CodeStyle/2.0.0-rc.3) provides better compatibility with C# 9 features. Usage: https://github.com/messerli-informatik-ag/code-style

Reply:
Addressed in 0970d02e67cdc9ea2a2de6ed4f19eab51fcb0482